### PR TITLE
Right Click Menus: remove unneeded actions

### PR DESCRIPTION
**Pull request by samwhale**
## Context

Had a meeting today with Andrew to discuss UX for the right click menu. Some actions were deemed confusing or unnecessary. This PR removes them.

[Doc](https://docs.google.com/document/d/1gcVTVMdGaM9Ds8T_EJAGQbiBlSArnJh_onuTBXmSISs/edit#heading=h.f930w6zw8ak)

## Summary

1. **Removing:**
- `Copy`
- `Paste`
- `Delete`
- `Replace Background Image`

And all tests associated with the above actions.

2. **Removes the page manipulation actions from the background media menu but not the page menu.** 

The page menu has nothing else, so I thought it would be a very sad empty menu without them.

3. Removes `lockAspectRatio` from the styles that can be copied, pasted, and cleared.

## Relevant Technical Choices

n/a

## To-do

n/a

## User-facing changes

||Text|Media|Background Media|Shapes|Page|
|--|--|--|--|--|--|
|Before|image|image|image|image|image|
|After|image|image|image|image|image|

## Testing Instructions

1. Turn on checklist flag
2. Go to editor
3. Try the right click menus on all the element types. Everything should work the same but some actions will not be there anymore. Should match the screenshots above.

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #

### CODE REVIEW

**Review by BrittanyIRL — COMMENTED** [review]
Thanks for doing this follow up so quickly! 

So, this is my bad for not realizing it yesterday - but since the page menu and the background image menu operate in the same space (so if there's a bg image you'll never get the "page" menu) it's a little jarring to have the options be completely different as with this update to remove the add/remove/duplicate page actions on bg image but not the page. 

Thoughts on leaving those actions for the bg image so that it's got more continuity with the page? 

Again, sorry for the late thought.

**Comment by samwhale** [comment]
> but since the page menu and the background image menu operate in the same space (so if there's a bg image you'll never get the "page" menu) it's a little jarring to have the options be completely different as with this update to remove the add/remove/duplicate page actions on bg image but not the page.
> 
> Thoughts on leaving those actions for the bg image so that it's got more continuity with the page?

Sure we could do that! Honestly I think we should combine them into one menu. If there is no background image, we could disable the actions that are background-media specific

**Review by maxyinger — APPROVED** [review]

File: packages/story-editor/src/app/rightClickMenu/test/useRightClickMenu.js [hunk 1]
```diff
@@ -139,21 +133,47 @@ describe('useRightClickMenu', () => {
...
+        RIGHT_CLICK_MENU_LABELS.SCALE_AND_CROP_BACKGROUND,
+        RIGHT_CLICK_MENU_LABELS.CLEAR_STYLE,
         RIGHT_CLICK_MENU_LABELS.ADD_NEW_PAGE_AFTER,
         RIGHT_CLICK_MENU_LABELS.ADD_NEW_PAGE_BEFORE,
         RIGHT_CLICK_MENU_LABELS.DUPLICATE_PAGE,
         RIGHT_CLICK_MENU_LABELS.DELETE_PAGE,
       ]);
     });
 
+    it('background media actions should be disabled', () => {
```
Comment (BrittanyIRL) [inline packages/story-editor/src/app/rightClickMenu/test/useRightClickMenu.js:168]: 👍

**Review by BrittanyIRL — APPROVED** [review]
Thanks for the update!

**Pull request merged**